### ISSUE TITLE
Please use WinGet Releaser

### ISSUE THREAD

**Issue by ppvnf** [JosefNemec/Playnite#3138]
### Feature description

Playnite is already available on Winget (Playnite.Playnite) and it seems up to date, but would it be possible to use [WinGet Releaser](https://github.com/marketplace/actions/winget-releaser) to make future versions automatically available on Winget? 

(As of yet, Winget does not have the ability to carry out such automations natively: https://github.com/microsoft/winget-pkgs/issues/500)

Thanks to [vedantmgoyal2009](https://github.com/vedantmgoyal2009) efforts, Playnite is automatically updated on Winget, but this isn't ideal because:

- Sometimes automation may go offline for maintenance, hence it is better to use GitHub Action. It is also easier to maintain and implement new features in the action. All apps use the latest manifest schema and fields.

- Automation is generally preferred for non-GitHub packages (meaning: software which are hosted on GitHub) that cannot take advantage of the GitHub Action.

- Since all GitHub API requests are authenticated with my account, using action distrubutes the amount of traffic on GitHub. The action is triggered automatically on releases, so we don't need to fetch APIs or scrape webpages to check for updates periodically, and PRs are created with the software's developer account.

- Microsoft prefers publishers managing releases for their apps themselves instead of a third party. It isn't a hard-and-fast rule, note the word "prefers" here.

**Comment by flyingdog1310** [comment]
Hi, sorry to bring this up again.

I just wanted to briefly mention that Winget's manifest schema has stabilized a lot, which makes the maintenance overhead much more manageable now. I've submitted Playnite Winget PRs [manually](https://github.com/microsoft/winget-pkgs/issues?q=is%3Aclosed%20is%3Apr%20author%3Aflyingdog1310) and I'm a DevOps engineer, so I'm confident I could write and maintain a solid automated release workflow — at no extra effort.

A couple of honest caveats worth noting upfront:
- `winget upgrade` won't handle breaking changes between versions config
- Upgrades via winget can fail if older version of Playnite is running at the time

Both are manageable, but I want to be transparent rather than oversell this. 

On a personal note — I manage pretty much all my software through the CLI, so having Playnite installable via winget would genuinely make my day. I imagine there are a few others like me out there. If you're open to it, I'm happy to invest the time. If not, I completely understand.

**Comment by JosefNemec** [comment]
I have no plans supporting WinGet going into to the future. On top of some missing features that are imo necessary, like support for differential/delta updates, I have ethical issue supporting WinGet as a platform (look up what happen to AppGet).

**Comment by flyingdog1310** [comment]
Fair enough. I wasn't aware of the AppGet situation. After looking it up, I get why that's a hard no. Delta update support is a fair point too. Thanks for explaining, appreciate it🙏.

**Comment by Wallby** [comment]
@JosefNemec The developer of AppGet has stated that they do not wish for people to not adopt WinGet.
Remove the space in the below URLs (to prevent GitHub notification system triggering):
github.com/microsoft/winget-cli/issues/ 353#issuecomment-635514407
github.com/microsoft/winget-cli/issues /353#issuecomment-637016016
